### PR TITLE
Octane features for 5.5.3-beta \ 5.6 release

### PR DESCRIPTION
**Pull request by m-seldin**
- jenkins issue #54253 - fix SSC (Fortify) integration 
- jenkins issue #54301 - silently close UFT dispatcher log 
- user story #733056: Jenkins Get status request should work without any existing octane configuration
job config (Jenkins ISSUE-54294)
- defect #734113 : Jenkins plugin - the jenkins user field descriptor (JENKINS ISSUE-54360)

https://issues.jenkins-ci.org/browse/JENKINS-54183

### CODE REVIEW

File: src/main/java/com/microfocus/application/automation/tools/octane/actions/WebhookExpectationAction.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package com.microfocus.application.automation.tools.model;
+package com.microfocus.application.automation.tools.octane.actions;
```
Comment (grount) [inline src/main/java/com/microfocus/application/automation/tools/octane/actions/WebhookExpectationAction.java]: If this class was serialized and introduced in previous versions, this change might break their configurations.

File: src/main/java/com/microfocus/application/automation/tools/octane/configuration/SSCServerConfigUtil.java [hunk 2]
```diff
@@ -17,58 +16,86 @@
...
-    }
+	private static String getFieldValue(Object someObject, String fieldName) {
+		for (Field field : someObject.getClass().getDeclaredFields()) {
+			field.setAccessible(true);
+			if (field.getName().equals(fieldName)) {
+				Object value = null;
+				try {
+					value = field.get(someObject);
+				} catch (IllegalAccessException e) {
+					e.printStackTrace();
```
Comment (grount) [inline src/main/java/com/microfocus/application/automation/tools/octane/configuration/SSCServerConfigUtil.java]: Shouldn't you use logger instead?

File: src/main/java/com/microfocus/application/automation/tools/octane/model/SonarHelper.java [hunk 3]
```diff
@@ -18,9 +18,8 @@
  * ___________________________________________________________________
  */
 
-package com.microfocus.application.automation.tools.model;
+package com.microfocus.application.automation.tools.octane.model;
```
Comment (grount) [inline src/main/java/com/microfocus/application/automation/tools/octane/model/SonarHelper.java]: If this class was serialized and introduced in previous versions, this change might break their configurations.

File: src/main/java/com/microfocus/application/automation/tools/octane/actions/WebhookExpectationAction.java (hunk 1 above)
Comment (gullerya) [reply src/main/java/com/microfocus/application/automation/tools/octane/actions/WebhookExpectationAction.java]: This class is new for this version.

File: src/main/java/com/microfocus/application/automation/tools/octane/model/SonarHelper.java (hunk 3 above)
Comment (gullerya) [reply src/main/java/com/microfocus/application/automation/tools/octane/model/SonarHelper.java]: This class is new for this version.

File: src/main/java/com/microfocus/application/automation/tools/octane/actions/WebhookExpectationAction.java (hunk 1 above)
Comment (gullerya) [reply src/main/java/com/microfocus/application/automation/tools/octane/actions/WebhookExpectationAction.java]: ... and it is NOT serialized.

File: src/main/java/com/microfocus/application/automation/tools/octane/model/SonarHelper.java (hunk 3 above)
Comment (gullerya) [reply src/main/java/com/microfocus/application/automation/tools/octane/model/SonarHelper.java]: ... and it is NOT serializable as well.

**Comment by grount** [comment]
Approved for test build

**Pull request merged**